tests/test_ingress.py

```diff
@@ -136,19 +151,19 @@ def test_post_with_raw_content_type_succeeds(self):
 
     def test_post_with_no_content_succeeds(self):
         count_before = IngressQueue.objects.count()
-        response = self.client.post(self.URL, '', content_type='raw')
+        response = self.client.post(self.URL, '', **AUTHORIZATION_HEADER, content_type='raw')
```

Comment:
Waarom stuur je expliciet `content_type='raw'` in deze test? Is dat omdat de client anders al voor het sturen een fout geeft over foute syntax?

Reply:
Ja klopt. Als je geen raw mee geeft dan denk Django dat het een form is en dus een lijst van key/value's heeft. Vervolgens faalt hij met een AttributeError: 'str' object has no attribute 'items' in de django client.